libs/product/src/models/composite-product-item.ts

```diff
@@ -1,10 +1,19 @@
 import { DaffProduct } from './product';
 
+/**
+ * An enum for whether a composite product item should be displayed as a radio or select input.
+ */
 export enum DaffCompositeProductItemInputEnum {
 	select = 'select',
 	radio = 'radio'
 }
 
+/**
+ * The composite product item describes one set of product options that the user can add to the composite product.
+ * If a composite product is a toolbox bundle, a composite product item might be a screw driver, and the options contained in that
+ * item might be a phillips head and a flathead. The customer would choose to add either a phillips head or a flathead to the
+ * composite product, or neither if the item is optional.
+ */
```

Comment:
I think this is a slightly better:

```ts
/**
 * The composite product item describes one set of product options that the user can add to the composite product.
 * A composite product item can be required or optional. If it is required, an option _must_ be chosed in order to add the product to the cart.
 * If the item is optional, the product can be added to the cart without an option having been chosen.
 * For example, if the composite product in question is a toolbox bundle, a composite product item could be a screw driver.
 * The options contained in that item could therefore be a phillips head and a flathead.
 * The customer could choose to add either a phillips head or a flathead to the composite product.
 */
```

Double check to make sure my descriptions are accurate.

Reply:
I took the first part added about optional vs required, but left the rest as it was